std_msgs/Header header
ArucoMarker[] marker_list
float32 camera_height